src/RemoteTech/Modules/ModuleRTDataTransmitter.cs

```diff
@@ -122,6 +122,7 @@ private IEnumerator Transmit(Callback callback = null)
                     }
                     yield return new WaitForSeconds(PacketInterval);
                 }
+                GameEvents.OnTriggeredDataTransmission.Fire(scienceData, vessel);
```

Comment:
Tried both ways, 
`GameEvents.OnTriggeredDataTransmission.Fire(scienceData, vessel, true);`
and
`GameEvents.OnTriggeredDataTransmission.Fire(scienceData, vessel, false);`
it doesn't fix MPL bug described #511